src/server/cdn-controller.js

```diff
@@ -30,16 +31,17 @@ class CdnController {
     });
 
     router.post('/api/v1/cdn/objects', [needsPermission(permissions.CREATE_FILE), multipartParser.array('files'), validateBody(postObjectsBodySchema)], async (req, res) => {
+      const utcNowString = moment.utc().toISOString();
       if (req.files && req.files.length) {
         const uploads = req.files.map(async file => {
           const fileName = req.body.prefix ? urls.concatParts(req.body.prefix, file.originalname) : file.originalname;
           const uniqueFileName = fileNameHelper.makeUnique(fileName);
-          await this.cdn.uploadObject(uniqueFileName, file.path, {});
+          await this.cdn.uploadObject(uniqueFileName, file.path, { createdon: utcNowString });
         });
         await Promise.all(uploads);
       } else if (req.body.prefix && req.body.prefix[req.body.prefix.length - 1] === '/') {
         // If no file but a prefix ending with `/` is provided, create a folder instead of a file:
-        await this.cdn.uploadEmptyObject(req.body.prefix, {});
+        await this.cdn.uploadEmptyObject(req.body.prefix, { createdon: utcNowString });
```

Comment:
Setting this could even go into the lower layer, in the `cdn` module, since it acts as a timestamp for us. For me that would be a better place, less error prone too.

Reply:
Done, thanks.